packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart

```diff
@@ -267,7 +267,7 @@ class _AtOnboardingActivateScreenState
       } else if (e == AtOnboardingResponseStatus.serverNotReached) {
         await _showAlertDialog(
           e,
-          title: AtOnboardingLocalizations.current.error_server_not_found,
+          title: AtOnboardingLocalizations.current.msg_atSign_unreachable,
         );
       } else if (e == AtOnboardingResponseStatus.timeOut) {
         await _showAlertDialog(
```

Comment:
Is title_session_expired the right message to show here if there was a timeout?

Reply:
Thanks for pointing it, fixed.